packages/sync/src/class-actions.php

```diff
@@ -260,18 +260,20 @@ public static function set_is_importing_true() {
 	 * @param string $queue_id               The queue the action belongs to, sync or full_sync.
 	 * @param float  $checkout_duration      Time spent retrieving queue items from the DB.
 	 * @param float  $preprocess_duration    Time spent converting queue items into data to send.
+	 * @param int    $queue_size             The size of the sync queue at the time of processing.
 	 * @return Jetpack_Error|mixed|WP_Error  The result of the sending request.
 	 */
-	public static function send_data( $data, $codec_name, $sent_timestamp, $queue_id, $checkout_duration, $preprocess_duration ) {
+	public static function send_data( $data, $codec_name, $sent_timestamp, $queue_id, $checkout_duration, $preprocess_duration, $queue_size = 0 ) {
```

Comment:
I don't think we should have 0 as the default state. Having _null_ as the default will make it so that queue_size isn't recorded as 0 in the site-profiles index for older versions of jetpack. Yes we could filter by Jetpack version but null is a better representation of the data.